augur/metadata_file.py

```diff
@@ -0,0 +1,89 @@
+import functools
+from os import access, R_OK
```

Comment:
I think we can delete this import along with the `check_file` method below.

Reply:
Removed.